packages/@blockprotocol/graph/src/graph-block-handler.ts

```diff
@@ -68,6 +68,20 @@ export class GraphBlockHandler
     return {};
   }
 
+  /**
+   * Registers multiple callbacks at once.
+   * Useful for bulk updates to callbacks after the service is first initialised.
+   */
+  registerCallbacks(callbacks: Partial<BlockGraphMessageCallbacks>) {
+    super.registerCallbacks(callbacks);
```

Comment:
What's the point of overriding a parent function when all you do is call it?

Reply:
For narrower typing. This could be achieved with generics, and I just started to do it, but I think it's going to take time disproportionate to its benefit to do properly